rhba-dev-env/ansible/roles/build_images/tasks/main.yml

```diff
@@ -0,0 +1,326 @@
+---
+- name: kinit red hat kerberos
+  shell: echo "{{ kerberos_password }}" | kinit {{ kerberos_username }}@REDHAT.COM
+
+- name: create cache_hack script
+  template:
+    src: "cache-hack.j2"
+    dest: /tmp/cache-hack.sh
+
+- name: run cache artifacts
+  shell: sh /tmp/cache-hack.sh
+
+- name: create build_overrides script
+  template:
+    src: "build-overrides.j2"
+    dest: /tmp/build-overrides.sh
+
+- name: run build-overrides {{ build_overrides_product }}  {{build_overrides_version}} {{ build_overrides_nightly }}
+  command: sh /tmp/build-overrides.sh -v {{build_overrides_version}} -p {{ build_overrides_product }}  -b {{ build_overrides_nightly }}
+
+- set_fact: var_build_product="{{ build_product }}" 
+            var_build_overrides_product="{{ build_overrides_product }}" 
+            var_rhpam="rhpam" 
+            var_rhdm="rhdm"
+            var_rhpam_all="rhpam-all" 
+            var_rhdm_all="rhdm-all" 
+            var_rhpam_businesscentral="rhpam-businesscentral"
+            var_rhpam_businesscentral_monitoring="rhpam-businesscentral-monitoring"
+            var_rhpam_businesscentral_indexing="rhpam-businesscentral-indexing"
```

Comment:
can be removed.

Reply:
Yup; not more indexing, as you mentioned above.